docs/collecting-data/code-generation/snowtype-config/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Snowtype configuration options"
+sidebar_position: 3
+---
+
+The Snowtype CLI configuration will be saved in a`snowtype.config.json` file after initialization. **We highly recommend you keep this file in the root of your project folder.**
+
+## Attributes in `snowtype.config.json`
+
+### `igluCentralSchemas`
+
+The schema tracking URLs from schemas available in [Iglu Central](http://iglucentral.com/).
```

Comment:
```suggestion
The URLs for schemas available in [Iglu Central](http://iglucentral.com/).
```

Reply:
Would keep the `schema tracking URLs` since it is the title of the value we need.

Reply:
where's that title used? I haven't seen it before. It feels hard to parse. Like, what is a `tracking URL`? Does this sentence mean "the schema that tracks URLs from schemas in Iglu Central"??
I would just call them `schema URIs`

Reply:
<img width="560" alt="image" src="https://github.com/snowplow/documentation/assets/15251081/8d694412-9418-44fa-b811-d3afd3381c2d">

Reply:
Oh, in that case, let's add a labelled screenshot like this to the page

Reply:
Or put `schema tracking URLs` as inline code